src/containers/Tenant/Diagnostics/TopShards/getTopShardsColumns.tsx

```diff
@@ -83,9 +91,19 @@ const tabletIdColumn: Column<KeyValueRow> = {
             return '–';
         }
         return (
-            <InternalLink to={createHref(routes.tablet, {id: row.TabletId})}>
-                {row.TabletId}
-            </InternalLink>
+            <React.Fragment>
+                <InternalLink to={createHref(routes.tablet, {id: row.TabletId})}>
+                    {row.TabletId}
+                </InternalLink>
+                <Button
+                    size="s"
+                    href={`${backend}/tablets?TabletID=${row.TabletId}`}
+                    className={b('external-button')}
+                    target="_blank"
+                >
+                    <Icon data={ArrowUpRightFromSquare} />
+                </Button>
```

Comment:
It looks like some `DeveloperUiLink` component will be helpful not to add hide and show styles to all tables. It could be done like `MonitoringButton`: https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/components/MonitoringButton/MonitoringButton.tsx